sci-visualization/spyview/spyview-20150124.ebuild

```diff
@@ -29,6 +29,8 @@ DEPEND="${COMMON_DEPEND}
 RDEPEND="${COMMON_DEPEND}
 	sci-visualization/gnuplot"
 
+PATCHES=( ${FILESDIR}/${P}-gnuplot_interface_fix.patch )
+
```

Comment:
try removing `src_configure()`, all those flags shouldn't be necessary in EAPI 6 anymore